at_commons/lib/src/keystore/at_key.dart

```diff
@@ -62,6 +74,10 @@ class AtKey {
     if (_sharedWith != null && _sharedWith!.isNotEmpty) {
       return '$_sharedWith:$key${_dotNamespaceIfPresent()}$_sharedBy';
     }
+    // if key starts with local: or isLocal set to true, return local key
+    if (key!.startsWith('local:') || (isLocal == true)) {
```

Comment:
I'm sure this works but it points to a more general problem with the code in this file. 

For example if the key actually starts with 'local:' - for example, `local:foo.test@alice` then the return value will be 'local:local:foo.test@alice"

Reply:
Updated to code to check for 'local:' in the AtKey.key